lib/events/eventrelation.h

```diff
@@ -17,9 +17,9 @@ struct QUOTIENT_API EventRelation {
     QString eventId;
     QString key = {}; // Only used for m.annotation for now
 
-    static constexpr auto ReplyType = "m.in_reply_to"_ls;
-    static constexpr auto AnnotationType = "m.annotation"_ls;
-    static constexpr auto ReplacementType = "m.replace"_ls;
+    static constexpr inline auto ReplyType = "m.in_reply_to"_ls;
```

Comment:
Aren't these, as class members, inlined automatically at least?..

Reply:
Class members are treated differently indeed, and it does look like they might actually be part of the ABI (`nm` shows them as `u` - unique global symbol). `inline` doesn't change that though either way, so I've removed that part of the change again.